src/offerings.ts

```diff
@@ -242,9 +242,12 @@ export interface PurchasesOffering {
      */
     readonly serverDescription: string;
     /**
-     * Offering metadata defined in RevenueCat dashboard.
+     * Offering metadata defined in RevenueCat dashboard. To access values, you need
+     * to check the type beforehand. For example:
+     * const my_unknown_value: unknown = offering.metadata['my_key'];
+     * const my_string_value: string | undefined = typeof(my_unknown_value) === 'string' ? my_unknown_value : undefined;
```

Comment:
I'm not sure if this is the best location for an example, but seems like it would be useful. FWIW, This code works for me.

Reply:
I think it's useful!